mx.sulong/libs/pthread.h

```diff
@@ -0,0 +1,15 @@
+#ifndef _PTHREAD_H
```

Comment:
I don't like it too, but we need the header file because `pthread_exit` usually has a `noreturn` attribute.
So, code compiled with the original header file won't run because it has an unreachable instruction after the `pthread_exit` call.

I wasn't sure where to put the header file, maybe we should move it to `include` instead of `mx.sulong/libs`?